components/markets/market-card/index.tsx

```diff
@@ -148,21 +152,26 @@ const MarketCardDetails = ({
         </span>
       </div>
       <div className="flex gap-2.5 text-sm">
-        {/* TODO: add market particpants and liquidity once added to indexer */}
-        {/* <div className="flex items-center gap-2">
+        {rows.numParticipants != null && (
+          <div className="flex items-center gap-2">
             <Users size={18} />
-            <span>223</span>
-          </div> */}
+            <span>{rows.numParticipants}</span>
+          </div>
+        )}
         <div className="flex items-center gap-2">
           <BarChart2 size={18} />
           <span>
             {formatNumberCompact(rows.volume)} {rows.baseAsset}
           </span>
         </div>
-        {/* <div className="flex items-center gap-2">
+        {rows.liquidity != null && (
+          <div className="flex items-center gap-2">
             <Droplet size={18} />
-            <span>223K ZTG</span>
-          </div> */}
+            <span>
+              {new Decimal(rows.liquidity).div(ZTG).toFixed(2)} {rows.baseAsset}
```

Comment:
Are you able to use `formatNumberCompact` util function to keep the number formatting consistent?